### PR TITLE
Delete data

### PR DESCRIPTION
**Pull request by xgaia**
- adding a button to clear the database in the stats tab
- escape underscores in devdoc 
- adding ttl directory in the .gitignore

### CODE REVIEW

**Comment by ofilangi** [comment]
I didn't see a message to confirm the deletion of database (User Interface)...can you add it ?

**Comment by xgaia** [comment]
- Confirm the deletion of database (yes/no)
- Functions to show/hide the modal box with custom message and button

**Comment by ofilangi** [comment]
+1

File: askomics/libaskomics/rdfdb/QueryLauncher.py [hunk 1]
```diff
@@ -98,6 +98,10 @@ def parse_results(self, json_res):
 
     def process_query(self, query):
         json_query = self.execute_query(query, log_raw_results=False)
+
+        if 'CLEAR GRAPH' in query:
```
Comment (abretaud) [inline askomics/libaskomics/rdfdb/QueryLauncher.py]: I'm not very happy with this if block, what happens if you go on with the parse_results method?

Comment (xgaia) [inline askomics/libaskomics/rdfdb/QueryLauncher.py]: The function process_query return an error

**Comment by xgaia** [comment]
Now, it delete database with the execute_query function, so it can't parse the results and don't raise an error

**Comment by abretaud** [comment]
Good, thanks! I finish to look at the rest as soon as possible

File: askomics/static/js/askomics.js [hunk 2]
```diff
@@ -110,12 +115,51 @@ function loadStatistics() {
...
+        hideModal();
+    };
     $('#content_statistics').append(table);
 
   });
 }
 
+function emptyDatabase(value) {
+    if (!value) {
+        no = 'no';
```
Comment (abretaud) [inline askomics/static/js/askomics.js]: I think 'no' is not used, am I right?

**Comment by xgaia** [comment]
Ok i'm done now

**Comment by abretaud** [comment]
Wonderful thank you!

**Pull request merged**